### PR TITLE
chore: adds fields to user for migration

### PR DESCRIPTION
**Pull request by markcsims**
Create new fields in User schema:

Skills = expertiseandResourceTags
SkillsDescription = expertiseandResourceDescription
approach = mainResearchInterests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -72,6 +72,38 @@ Individual `packages` or `apps` may contain their own readme files as deemed nec
 The [`docs`](docs) folder contains overall architecture / decision documentation.
 The [`dev`](dev) folder contains documentation regarding the local dev environment for the backend.
 
+## Squidex schema changes and graphql
+
+Once schema changes have added via packages/squidex/schema and released. The GraphQL definations need to be updated.
```
Comment (peterstarling) [inline README.md:77]:
```suggestion
Once schema changes are added via packages/squidex/schema and released the GraphQL definitions need to be updated. 
```

File: README.md [hunk 2]
```diff
@@ -72,6 +72,38 @@ Individual `packages` or `apps` may contain their own readme files as deemed nec
...
+
+`yarn schema:update`
+
+Then commit the changed.
```
Comment (peterstarling) [inline README.md:81]: we should say that the command updates the schema and queries according to the graphql schema of the application that your environment is pointing to (through the .env file for example)

File: apps/asap-server/src/migrations/1636027226-move-repurposed-fields.ts [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  };
+  down = async (): Promise<void> => {
+    /* Ignore  */
+  };
+}
+async function migrateUserFields() {
+  await applyToAllItemsInCollection<RestUser>(
+    'users',
+    async (
+      {
```
Comment (peterstarling) [inline apps/asap-server/src/migrations/1636027226-move-repurposed-fields.ts:19]: personally i find destructuring such complex data structures a little hard to read, would you be better off using a simple variable here and the referencing the props via dot notation?

File: apps/asap-server/src/migrations/1636027226-move-repurposed-fields.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
+/* istanbul ignore file */
+import { RestTeam, RestUser } from '@asap-hub/squidex';
+import { Migration } from '../handlers/webhooks/webhook-run-migrations';
+import { applyToAllItemsInCollection } from '../utils/migrations';
+
+export default class MoveRepurposedFields extends Migration {
+  up = async (): Promise<void> => {
+    await migrateUserFields();
+    await migrateTeamFields();
```
Comment (peterstarling) [inline apps/asap-server/src/migrations/1636027226-move-repurposed-fields.ts:9]: maybe break it into two separate migrations?

**Review by peterstarling — CHANGES_REQUESTED** [review]

File: apps/asap-server/src/migrations/1636027226-move-repurposed-fields.ts (hunk 4 above)
Comment (markcsims) [reply apps/asap-server/src/migrations/1636027226-move-repurposed-fields.ts:9]: agreed to leave as is

File: README.md (hunk 2 above)
Comment (markcsims) [reply README.md:81]: done

File: apps/asap-server/src/migrations/1636027226-move-repurposed-fields.ts (hunk 3 above)
Comment (markcsims) [reply apps/asap-server/src/migrations/1636027226-move-repurposed-fields.ts]: Done

File: README.md (hunk 1 above)
Comment (markcsims) [reply README.md]: committed changes

**Pull request merged**